ndk/src/main/jni/utils/stack_unwinder_libunwindstack.cpp

```diff
@@ -0,0 +1,68 @@
+#include "../utils/string.h"
+#include "stack_unwinder_libunwindstack.h"
+#include <stdlib.h>
+#include <ucontext.h>
+#include <unwindstack/Elf.h>
+#include <unwindstack/MapInfo.h>
+#include <unwindstack/Maps.h>
+#include <unwindstack/Memory.h>
+#include <unwindstack/Regs.h>
+
+ssize_t
+bsg_unwind_stack_libunwindstack(bsg_stackframe stacktrace[BUGSNAG_FRAMES_MAX],
+                                siginfo_t *info, void *user_context) {
+  if (user_context == NULL) {
+    return 0; // only handle unwinding from signals
+  }
+
+  const std::unique_ptr<unwindstack::Regs> regs(
+      unwindstack::Regs::CreateFromUcontext(unwindstack::Regs::CurrentArch(),
+                                            user_context));
+
+  // if (user_context != NULL) {
```

Comment:
This can probably be removed and accessed from VCS if we need some record of it.